tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-card.html

```diff
@@ -242,17 +253,29 @@
       },
       _csvUrl(tag, run) {
         return tf_backend.addParams(
-            this.getDataLoadUrl({tag, run}),
+            this._getDataLoadUrl({tag, run}),
             {format: 'csv'});
       },
       _jsonUrl(tag, run) {
-        return this.getDataLoadUrl({tag, run});
+        return this._getDataLoadUrl({tag, run});
       },
       _getDataSeries() {
-        return this.dataToLoad.map(this._getSeriesNameFromDatum);
+        return this.dataToLoad.map(d => this._getSeriesNameFromDatum(d));
       },
       _getSeriesNameFromDatum(datum) {
-        return datum.run;
+        return  `${datum.experiment || 'default'}/${datum.run}`;
```

Comment:
Optional, but I might consider just using the empty string instead of "default".  That way it's guaranteed not to collide with any actual experiment name.

Also, maybe worth adding a TODO to adjust the tooltip generation so this isn't needed, just to be clear that this is only a v0 solution.